src/v/cluster/persisted_stm.cc

```diff
@@ -60,7 +60,7 @@ ss::future<std::optional<stm_snapshot>> persisted_stm::load_snapshot() {
 
     if (version == snapshot_version_v0) {
         vlog(
-          clusterlog.warn,
+          _log.warn,
```

Comment:
If we are here, we should probably use a prefix_logger instance (_ctx_log) from rm_stm for consistency? Every log line has a ntp={}.